src/pages/home/Search.scss

```diff
@@ -1,29 +1,28 @@
 .main {
-  @apply bg-center bg-cover flex items-center justify-center px-4 pb-12 pt-24;
-  height: 550px;
+  @apply bg-center bg-cover flex items-center justify-center px-4 py-8;
 }
 
 .inner-form {
-  @apply pt-16 pl-24;
-  position: center;
-  width: 1000px;
+  display: flex;
+  justify-items: center;
+  flex-direction: column;
 }
 
 .main-form {
-  @apply bg-black rounded-sm inline-block p-10 relative;
+  @apply bg-black rounded-sm p-10;
   box-shadow: 0 8px 20px 0 rgb(0 0 0 / 15%);
-  width: 800px;
```

Comment:
default form alignment for a large screen is changed because of this.. instead of changing all the current styles, you can keep your changes for a minimum width screen separately.